service/api_logic/news_logic.py

```diff
@@ -60,6 +64,17 @@ def json_news(self, news_records):
             content_type='application/json; charset=utf-8',
         )
 
+    def get_filtered_news(self, filters: NewsDTO, pagination: Pagination):
+        query = self._news_dal.get_query().order_by(desc(News.save_at))
```

Comment:
And what If I have a filter which would require older news? What if a user decides to read the oldest news we have?

This should be moved to a separate filter and applied respectively. Do this for all other places if I missed any, please